tests/test_basicio.py

```diff
@@ -28,7 +30,7 @@ def localserver(tmpdir_factory):
 
 @pytest.mark.skip("not implemented")
```

Comment:
Does this still need to be skipped now that the ping check is implemented?

Reply:
It doesnt pass test_broken_server(). It just waits and I don't know how long because I'm not willing to wait long enough

Reply:
Did you see https://github.com/xrootd/xrootd/issues/1735 ? You could wait 15s or I guess alter that environment variable to wait less time

Reply:
its taking much longer than 15 seconds for me